.github/workflows/go.yml

```diff
@@ -40,9 +31,6 @@ jobs:
     - name: Format
       run: if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then exit 1; fi
 
-    - name: Install dependencies
```

Comment:
I don't remember exactly but it might have been the case that this step allowed the cache to cache the dependencies better